hasura.planx.uk/migrations/default/1753098512705_update_service_charges_view_fiscal_year/up.sql

```diff
@@ -0,0 +1,26 @@
+DROP VIEW IF EXISTS "public"."service_charges";
+CREATE OR REPLACE VIEW "public"."service_charges" AS (
+    select 
+      ps.team_slug,
+      f.slug as flow_slug,
+      f.name as flow_name,
+      ps.session_id,
+      ps.payment_id,
+      (ps.fee_breakdown->'amount'->>'serviceCharge')::numeric as service_charge_amount,
+      ps.created_at as paid_at,
+      extract(month from ps.created_at) as month,
+      to_char(ps.created_at, 'Month') as month_text,
+      case 
+        when extract(quarter from ps.created_at) = 1 then 4
+        else extract(quarter from ps.created_at) - 1
+      end as fiscal_year_quarter,
+      case
+        -- UK Fiscal Year starts April 1, so Jan - March of current year are "last" fiscal year
+        when extract(month from ps.created_at) <= 3 then extract(year from ps.created_at) - 1
+        else extract(year from ps.created_at)
+      end as fiscal_year
```

Comment:
TODO - double-check if "FY 2025-26" is correctly categorised as "2025" (because most months of this fiscal year start/fall within this year) or if it should actually be "2026" (because this is when the fiscal year ends)